packages/api/src/resolvers/article/index.ts

```diff
@@ -275,114 +275,6 @@ export const createArticleResolver = authorized<
         }
       }
 
-      const existingArticle = await models.article.getByUrlAndHash({
-        url: articleToSave.url,
-        hash: articleToSave.hash,
-      })
```

Comment:
Checking of existing article has been removed because in elastic we store page and link in the same entity